magento/magento2
# Update success message on admin backend with field changed ref

### Description (*)

In relation to this issue https://github.com/magento/magento2/issues/38352, it was detected that the admin user when modifying some field of his account and recording it, the message he received was generic (You saved the account).

For this reason, through this PR we add the information of the modified field(s) to the success message.


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#38352

### Manual testing scenarios (*)

1. Backend -> account setting
2. modify 1 field and save.
3. modify 2 fields and save.
4. modify x fields and save.
5. Verify that all fields can be modified

Attached video of the tests
[successMessage.webm](https://github.com/magento/magento2/assets/115582581/930d6c5d-1da7-435a-8c8f-adf6302a8f6b)

![image](https://github.com/magento/magento2/assets/115582581/f0a605ac-ec6c-411d-97d3-4c35b5d05899)
![image](https://github.com/magento/magento2/assets/115582581/7345a642-c5fb-4062-a57f-d88ea089a542)
![image](https://github.com/magento/magento2/assets/115582581/0fa092d7-bbf1-4bbb-a70c-2dc4a0202790)

### Questions or comments

FYI: Redirecting to the main dashboard was considered but since the user can decide to generate other modifications after modifying the password, that feature was rejected.


### Contribution checklist (*)
